### PR TITLE
[fix](auth)Stream load not have usage priv for workload group

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

When requesting execution plans from FE through RPC, the client IP was previously used and changed to the matching user host.
For example, the previous version was' 82.157.43.14 ', and the later version after the modification was' 82.157.43%',
FE will directly build userIdentity using the brought IP.
Due to the possibility that only user a @'82.157.43.% 'may exist on FE and not a @'82.157.43.14', the user may not have permission.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

File: be/src/runtime/stream_load/stream_load_executor.cpp [hunk 1]
```diff
@@ -207,6 +207,9 @@ Status StreamLoadExecutor::begin_txn(StreamLoadContext* ctx) {
         ctx->db_id = result.db_id;
     }
     ctx->need_rollback = true;
+    if (result.host != "") {
```
Comment (morningman) [inline be/src/runtime/stream_load/stream_load_executor.cpp:210]: check `result._isset.host`?

**Review by morningman — DISMISSED** [review]
LGTM

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Pull request closed without merging**